README.rst

```diff
@@ -704,7 +714,20 @@ If you need to use template tags of *openwisp_notifications*, you will need to r
 For more information about template tags in django, please refer to the
 `"Custom template tags and filters" section in the django documentation <https://docs.djangoproject.com/en/dev/topics/http/urls/>`_.
 
-15. Add Base Template for Admin
+15. Register Notification Types
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can register notification types as shown in the `section for registering notification types <#register_notification_type>`_.
+
+A reference for registering a notification type is also provided in
+`sample_notifications/apps.py <https://github.com/openwisp/openwisp-notifications/blob/master/tests/openwisp2/sample_notifications/apps.py>`_.
+The registered notification type of ``sample_notifications`` app is used for creating notifications
+when an object of ``TestApp`` model is created. You can use
+`sample_notifications/models.py <https://github.com/openwisp/openwisp-notifications/blob/master/tests/openwisp2/sample_notifications/models.py>`_
+as reference for your implementation.
+
```

Comment:
Remove this blank line please

Reply:
Okay, but it was added deliberately.  :grimacing: